shell.py

```diff
@@ -12,10 +14,30 @@ def write(lba, data):
     """write"""
     pass
 
-@cli.command(name="read")
-@click.argument('lba')
-def read(lba):
-    pass
+def call_system(cmd:str):
+    try:
+        result = subprocess.run("dir", shell=True, capture_output=True, text=True, encoding='cp949', check=True)  # or 'euc-kr'
```

Comment:
```suggestion
        result = subprocess.run(cmd, shell=True, capture_output=True, text=True, encoding='UTF-8', check=True)  # or 'euc-kr'
```
혹시 이렇게 바꿔야하는거 아닐지요?
그리도 encoding 방식이 일반적이지 않은것 같아서 `UTF-8` 이런걸로 바꿔야하는게 아닌가 생각이 드는데 혹시 `cp949`인 특별한 이유가있을까요?